reproman/resource/singularity.py

```diff
@@ -81,8 +91,8 @@ def create(self):
             # Start the container instance.
             # NOTE: Logging stdout and stderr hangs the run call, so we
             # disable the logging in the run call below.
-            self._runner.run(['singularity', 'instance.start', self.image,
-                self.name], log_stdout=False, log_stderr=False)
+            self._run_instance_command('start', [self.image, self.name],
+                log_stdout=False, log_stderr=False)
```

Comment:
I don't know what linter pycharm uses, but I would have guessed that it would yell at you about "continuation line under-indented for visual indent" (E128).

Reply:
ok, will address this and another one nearby... FWIW -- pycharm does scream ;-)